### ISSUE TITLE
Precompiled ipget-0.7.0 doesn't work on Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by vmx** [ipfs/ipget#100]
The precompiled ipget v0.7.0 doesn't work on Ubuntu 20.04 LTS, due to a too recent libc requirement (>=2.32), while Ubuntu 20.04 LTS only ships with libc 2.31. I think it would be great to support the current Ubuntu LTS releases.

A workaround is to use ipget v0.6.0.

**Comment by stdedos** [comment]
```console
$ ./ipget
./ipget: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./ipget)
$ ldd --version ldd
ldd (Ubuntu GLIBC 2.31-0ubuntu9.2) 2.31
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

pls compile with "compatible" GLIBC versions 😕

**Comment by vmx** [comment]
This is resolved with ipget v0.8.1. You can get it from https://dist.ipfs.io/ipget/v0.8.1.
